components/admin/requests/physician-assessment/Card.tsx

```diff
@@ -108,6 +170,41 @@ const Card: FC<Props> = props => {
               </GridItem>
             </>
           )}
+          {mobilityAids?.length !== 0 && (
+            <>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  Mobility aids
+                </Text>
+              </GridItem>
+              <GridItem>
+                <List>
+                  {mobilityAids?.map(aid => (
+                    <ListItem key={aid}>
+                      <Text as="p" textStyle="body-regular" textAlign="left">
+                        {titlecase(aid)}
+                      </Text>
+                    </ListItem>
+                  ))}
+                </List>
+              </GridItem>
+            </>
+          )}
+          {mobilityAids?.includes('OTHERS') && (
+            <>
+              {/* Other mobility aids description */}
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  Other mobility aids
+                </Text>
+              </GridItem>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  {otherMobilityAids}
+                </Text>
+              </GridItem>
+            </>
+          )}
```

Comment:
Confirming with designers if they want this (see slack thread https://uwblueprint.slack.com/archives/C021S4WGHBR/p1652067821964179)